docs/question/server.md

```diff
@@ -36,6 +36,18 @@ def generate(data):
     data["correct_answers"]["y"] = 2 * data["params"]["x"]
 ```
 
+The values set can then be used in the `question.html` file by using `{{params.NAME}}`. So for example, you can use `data["params"]["x"]` with `{{params.x}}`.
+
+The snippet below uses a [Mustache comment](./template.md#hiding-staff-comments-in-questionhtml), and the [`<pl-question-panel>`](./elements.md#pl-question-panel-element) element so that it is only shown within the context of the [question panel](./template.md#question-panel).
+
+<!-- prettier-ignore -->
+```html title="question.html"
+<pl-question-panel>
+  {{! The text of this question is only shown on the question panel. }}
```

Comment:
Is a Mustache comment really necessary to demonstrate here? This file is specifically documenting `server.py`. IMO this just clutters the example with unnecessary information.